qa/L0_analyze/test.sh

```diff
@@ -20,16 +20,17 @@ rm -rf results && mkdir -p results
 
 # Set test parameters
 MODEL_ANALYZER="`which model-analyzer`"
-MODEL_REPOSITORY=${MODEL_REPOSITORY:="/mnt/dldata/inferenceserver/model_analyzer_clara_pipelines"}
-QA_MODELS="`ls $MODEL_REPOSITORY | head -2`"
+REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
+MODEL_REPOSITORY=${MODEL_REPOSITORY:="/mnt/dldata/inferenceserver/$REPO_VERSION/libtorch_model_store"}
+QA_MODELS="vgg19_libtorch resnet50_libtorch"
 MODEL_NAMES="$(echo $QA_MODELS | sed 's/ /,/g')"
 BATCH_SIZES="1,2"
 CONCURRENCY="1,2"
 EXPORT_PATH="`pwd`/results"
 FILENAME_SERVER_ONLY="server-metrics.csv"
 FILENAME_INFERENCE_MODEL="model-metrics-inference.csv"
 FILENAME_GPU_MODEL="model-metrics-gpu.csv"
-TRITON_LAUNCH_MODE=${TRITON_LAUNCH_MODE:="docker"}
+TRITON_LAUNCH_MODE=${TRITON_LAUNCH_MODE:="local"}
```

Comment:
Is there a reason for this change?

Reply:
When using docker mode,  we will run 21.04 by default which will have problems but the local version is 21.03 for now. We can use docker too but we have to fix the container to 21.03.

Reply:
Ah okay